### ISSUE TITLE
Switch project disable status bar (not always)

### ISSUE THREAD

**Issue by PixelT** [jccguimaraes/atom-project-viewer#155]
Sometimes, when I switch between project list, the status bar will be disabled.
I must manually enable it (status-bar) in settings -> packages

**Comment by rgawenda** [comment]
It also disables linter and linter-ui-default for me when it happens

**Comment by jccguimaraes** [comment]
hey guys, there is a new option for disable/enable packages on project switch. please remove the default packages to solve this in the short term.

PS: are you all using the same instance window to switch projects?

thankz for reporting

**Comment by rgawenda** [comment]
Yes. Setting that value to 'none' seems to fix this for me.

**Comment by lneveu** [comment]
Hi! Same bug here (its took me days before realizing that project-viewer was the problem 😆 ) 
As you said @jccguimaraes, I've removed default values and it seems to fix the issue.

**Comment by jccguimaraes** [comment]
can you please tell me if you use same instance of atom to switch projects?
thanks

**Comment by rgawenda** [comment]
I only launch one instance of atom from operating environment. Then I usually open a different project in a new window from project viewer (up to four windows/projects).

**Comment by jccguimaraes** [comment]
ok, i think the problem is only happening when new windows are opened. I have restricted this feature to only apply on project switching within the same Atom window.

If any of you have a different scenario please inform me.

thanks for all the feedback on this

**Comment by Miguel-F** [comment]
I just noticed that my status bar was no longer showing and stumbled upon this post. Sure enough the status-bar package had been disabled. I used the 'none' option mentioned and so far so good.  I only bring this up because I do not launch different windows.  I only use one Atom window and had this issue.  I don't know if this is related or not but I have also noticed that when I open one project then switch to another project, I cannot switch back to the initial project that was opened.  Nothing appears to happen. I don't see any errors but the project does not open.  I have to close and reopen Atom to open the initial project again.

**Comment by rgawenda** [comment]
Try to open the first project again, I mean "twice", repeat the failing
step. And tell us if it works.

**Comment by Miguel-F** [comment]
Not sure what you mean by "repeat the failing step".  I just relaunched Atom and verified that I am still having that issue.  Let me try to explain again.  I have several projects defined within Project Viewer.  I open a project using Project Viewer.  I then open a different project.  If I try to select the initial project again it will not open.  If I close Atom and re-open Atom, the projects are accessible again (but only the first time).  The projects I have defined are not local to my machine, they are network paths like "//servername/sharename" (if that matters). This could definitely be my own environment issue but this did work before.

**Comment by rgawenda** [comment]
Instead of closing Atom, retry the step that's not working. I filed a bug
about something similar this week: if I try to open a project in a new
window nothing happens, but if I retry it works. Sorry about my poor English

**Comment by Miguel-F** [comment]
I don't really have anything to retry.  I am simply opening one project - say project A.  Then open another project - say project B.  Then try to open project A again but it will not open.  I am not doing anything between those steps.  Just opening projects.

Hmm???  Now I am seeing this issue when I try to switch projects - https://github.com/jccguimaraes/atom-project-viewer/issues/153.  I had not seen that before today.

**Comment by rgawenda** [comment]
Step 1: open A
Step 2: open B
Step 3: open A (fail)
Step 4: open A (goto step 3)

**Comment by Miguel-F** [comment]
Yes.  For me it was:
Step 1: open A (works)
Step 2: open B (works)
Step 3: open A (fails) - and it will continue to fail no matter how many times I try.

Closing and reopening Atom used to work but right now my projects are not opening at all.  I keep getting this error now https://github.com/jccguimaraes/atom-project-viewer/issues/153  It seems to be getting worse for me.

**Comment by jccguimaraes** [comment]
hey guys, sorry for making an update in a while. really up to the neck in work. but i am trying to fix what i can. i really hope this weekend i can throw a more stable version :(

**Comment by jccguimaraes** [comment]
can you please tell me if you are on beta channels? Atom is breaking a lot of code :eyeglasses: LOL

**Comment by Miguel-F** [comment]
No worries @jccguimaraes .  Thanks for providing/maintaining this package.
I am not on the Atom beta channel.  I am running Atom version 1.18.0 and Project Viewer version 1.1.2.
Unfortunately I am no longer able to open any of my projects at the moment.  I will work on this some more today and let you know what I find out.  In the meantime if you have any ideas for me please let me know.

**Comment by jccguimaraes** [comment]
@Miguel-F try to use `Project-viewer -> Toggle Select View` to see if at least you can switch with that option and if you would be so kind, inform me if it also breaks

**Comment by Miguel-F** [comment]
No that does not work either.  It is giving me a different error.

[Enter steps to reproduce:]

1. Click Project-Viewer -> Toggle Select View...
2. Select one of the projects

**Atom**: 1.18.0 ia32
**Electron**: 1.3.15
**OS**: Microsoft Windows 7 Enterprise 
**Thrown From**: [project-viewer](https://github.com/jccguimaraes/atom-project-viewer) package 1.1.2


### Stack Trace

Uncaught TypeError: Cannot set property 'treeView' of undefined

```
At F:\maf\Documents\Workspaces\Atom\.atom\packages\project-viewer\src\project-view.js:270

TypeError: Cannot set property 'treeView' of undefined
    at /packages/project-viewer/src/project-view.js:270:26)
    at SelectListView.confirmed (/packages/project-viewer/src/select-list-view.js:52:8)
    at /packages/project-viewer/src/select-list-view.js:64:10)
    at /packages/project-viewer/node_modules/atom-space-pen-views/lib/select-list-view.js:139:19)
    at HTMLOListElement.dispatch (/packages/project-viewer/node_modules/jquery/dist/jquery.js:4435:9)
    at HTMLOListElement.elemData.handle (/packages/project-viewer/node_modules/jquery/dist/jquery.js:4121:28)
```

### Commands

```
  2x -1:27.8.0 project-viewer:toggleSelectList (atom-workspace.workspace.scrollbars-visible-always.theme-one-light-syntax.theme-one-light-ui)
```

### Non-Core Packages

```
atom-beautify 0.30.3 
autocomplete-cfml 0.5.0 
language-cfml 0.17.0 
less-than-slash 0.16.0 
project-viewer 1.1.2 
split-diff 1.4.3 
```

**Comment by jccguimaraes** [comment]
thanks! will try to make it work again!

**Comment by Miguel-F** [comment]
@jccguimaraes thanks.  And to be honest, this could totally be my own environment setup.  I don't know that anyone else is experiencing my issues.  To let you know, I am using network drives for my Atom project folders and my Atom workspace.  This has been working for a while.  It did not break until I tried to get the status bar back by changing the project viewer setting.

**Comment by jccguimaraes** [comment]
thanks for a more constructed feedback. will take that in mind. :thumbs_up:

**Comment by Miguel-F** [comment]
@jccguimaraes here is a quick follow up.  I went back and set "List of packages to reload" to "none" in order to get the Atom status bar back.  Then I started getting this error https://github.com/jccguimaraes/atom-project-viewer/issues/152  In that thread the user mentioned turning off the "Keep Context" option. So I did that.  Now I am back to my original problem.  I can now open a project using project viewer but only the first time.  If I open another project and then go back to the initial project it will not open and I do not see any error message popup.  At least it is partially working again :smiley:

**Comment by jccguimaraes** [comment]
damn! what a loop :) thanks mate

**Comment by lneveu** [comment]
Hello! I noticed that you didn't publish a new version with my PR #152 (you only merged it). Maybe its can solve some issues that people here experienced?

**Comment by jccguimaraes** [comment]
hey @lneveu, did not have much time to add more stuff. sorry for that. `1.2.0` has your contribution! :)

**Comment by jccguimaraes** [comment]
please validate this issue!

**Comment by lneveu** [comment]
@jccguimaraes thx for your time! I'll check that!

**Comment by Miguel-F** [comment]
@jccguimaraes  thank you so much for taking the time to work on this issue!  After updating to the latest release I can now switch back and forth between my projects.  It is working again.  Thank you so much!!!!  I even went back and turned on "Keep Context" again (even though I don't fully understand what that option is for).  It is still working.  I believe your latest release has fixed all of the issues I was having.  I will continue to monitor and let you know.  Thanks again!

**Comment by jccguimaraes** [comment]
@Miguel-F and @lneveu thank you a lot for giving all the feedback!

**Comment by Hativ3** [comment]
The initial problem is not fixed for me. Packages are not reloaded, packages are disabled. Except I change the default to "none". I use the same instance, no new window.

Atom 1.19.2
project-viewer 1.2.0

**Comment by jccguimaraes** [comment]
hey @Hativ3 thanks. will look at it
